data/tutorials/getting-started/1_01_a_tour_of_ocaml.md

```diff
@@ -260,7 +260,7 @@ val range : int -> int -> int list = <fun>
 - : int list = [2; 3; 4; 5]
 ```
 
-As indicated by its type `int -> int -> int list`, the function `range` takes two integers as arguments and returns a list of integers as result. The first `int` parameter, `lo`, is the range's lower bound; the second `int` parameter, `hi`, is the higher bound. If `lo > hi`, the empty range is returned. That's the first branch of the `if … then … else` expression. Otherwise, the `lo` value is prepended to the list created by calling `range` itself; this is recursion. Preprending is achieved using `::`, the cons operator in OCaml. It constructs a new list by adding an element at the front of an existing list. Progress is made at each call; since `lo` has just been appended at the head of the list, `range` is called with `lo + 1`. This can be visualised this way:
+As indicated by its type `int -> int -> int list`, the function `range` takes two integers as arguments and returns a list of integers as result. The first `int` parameter, `lo`, is the range's lower bound; the second `int` parameter, `hi`, is the higher bound. If `lo > hi`, the empty range is returned. That's the first branch of the `if … then … else` expression. Otherwise, the `lo` value is prepended to the list created by calling `range` itself; this is recursion. Prepending is achieved using `::`, the cons operator in OCaml. It constructs a new list by adding an element at the front of an existing list. Progress is made at each call; since `lo` has just been prepended at the head of the list, `range` is called with `lo + 1`. This can be visualised this way:
```

Comment:
Same here. We should probably have either: 

> has been prepended to the list

or 

> has been appended at the head of the list

Reply:
Let's look at this section a little more closely. 

"Otherwise, the `lo` value is prepended to the list created by calling `range` itself; this is recursion."

This is trying to say that the `lo` value is added to the front of the list that was previously created by calling `range` --- correct? If so, I'll work on making that more clear. Is the "itself" necessary here?

"Prepending is achieved using `::`, the cons operator in OCaml. It constructs a new list by adding an element at the front of an existing list."

IIUC, this says: The OCaml cons operator (`::`) constructs a new list by adding an element at the front of an existing list" 

This is currently how it reads. The "It" in the second sentence refers to `::`, right?


Unless "prepending" is a specific term for this action in programming, it might be best to avoid it.